utils/src/main/kotlin/OkHttpClientHelper.kt

```diff
@@ -65,14 +65,15 @@ object OkHttpClientHelper {
             val specs = listOf(ConnectionSpec.MODERN_TLS, ConnectionSpec.COMPATIBLE_TLS, ConnectionSpec.CLEARTEXT)
 
             // OkHttp uses Java's global ProxySelector by default, but the Authenticator for a proxy needs to be set
-            // explicitly. Also note that no (non-proxy) authenticator is set here as it is primarily intended for
+            // explicitly. Also note that the (non-proxy) authenticator is set here and is primarily intended for
```

Comment:
Commit message: This change now does not only affect the Downloader anymore, but any user of `OkHttpClientHelper`. So maybe a better commit message:

```
OkHttpClientHelper: Support "reactive" authentication not only for proxies

In order to be able to download e.g. Maven artifacts from a private
repository that requires authentication, make our OkHttp client support
"reactive" authentication also for no-proxy hosts.
```
```

Reply:
done